extra/2-piping.js

```diff
@@ -16,26 +16,26 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a,b) {
+   return a + b
 }
 
-function multiply() {
-
+function multiply(a,b) {
+   return  parseFloat(a)* parseFloat(b)
 }
 
-function format() {
-
+function format(a) {
+   return `£${a}`
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = "£24";
```

Comment:
Hi Oleh, for this badCode, this is meant to be written out instead of just the answer in the code (2. Using the variable startingValue as input, perform the following operations using your functions all
  on one line (assign the result to the variable badCode):
  - add 10 to startingValue
  - multiply the result by 2
  - format it)

Reply:
Thank you for helping with that exercise Andriana. I fixed it.